contracts/cw20-staked-balance-voting/src/msg.rs

```diff
@@ -52,6 +60,7 @@ pub enum ExecuteMsg {}
 
 #[voting_query]
 #[token_query]
+#[active_query]
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum QueryMsg {
```

Comment:
How would you feel about adding a query here for the actual threshold being used as well as if they module is active?